### DISCUSSION TITLE
@graphql-codegen/client-preset-swc-plugin doesn't seem to do anything with nextjs

### DISCUSSION THREAD

**Discussion by CaptainN** [dotansimha/graphql-code-generator#9441] [General]
Hello! Thanks for all the work on these codegen tools, the typescript generation has already helped cache multiple errors in my codebase!

I did try to get @graphql-codegen/client-preset-swc-plugin working with the latest version of nextjs (13.4.2) and haven't been able to observe any bundle size benefits. The chunk that contains all the grpahql queries is exactly the same whether that's enabled or not. Could it be a configuration issue?

Here is my codegen config:

```
import type { CodegenConfig } from "@graphql-codegen/cli";

const config: CodegenConfig = {
  overwrite: true,
  schema: "./schema.graphql",
  documents: ["src/data/queries/**/*.ts"],
  generates: {
    "src/data/types/": {
      preset: "client",
    },
  },
};

export default config;
```

Here is my client-preset-swc-plugin config inside of next.config.js:

```
  experimental: {
    swcPlugins: [
      [
        "@graphql-codegen/client-preset-swc-plugin",
        { artifactDirectory: "./src/data/types", gqlTagName: "graphql" },
      ],
    ],
  },
```

Size with plugin enabled:
![image](https://github.com/dotansimha/graphql-code-generator/assets/245825/ee28f186-6859-4368-878d-1556f4e92201)

Size with plugin disabled:
![image](https://github.com/dotansimha/graphql-code-generator/assets/245825/779b66a6-715d-4e58-8b90-1dfa814bdbed)

Any help would be appreciated! Thanks!

**Comment by rik-iso** [comment]
Did you ever figure this out? I also don't seem to have any effect from it.

Using the latest forked version compatible with 14.2: https://www.npmjs.com/package/@benjibenji/graphql-codegen-client-preset-swc-plugin

**Comment by CaptainN** [comment]
No, and even if I use Babel, it still doesn't do much. I didn't try to change it out for a different lookupethos, but that wasn't compatible with apollow client (works with URQL though). I'll see if I can wrap so etching up.
